engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/number/utils/BigIntegerOps.java

```diff
@@ -235,16 +236,7 @@ public static boolean fitsInLong(BigInteger bigInteger) {
   }
 
   public static boolean fitsInLong(double decimal) {
```

Comment:
Given this is now a tiny wrapper on `Core_Math_Utils.fits_in_long` could we just use that?